tensorboard/plugins/debugger_v2/debug_data_multiplexer_test.py

```diff
@@ -34,11 +37,45 @@ def start(self):
 
 
 class DebuggerV2PluginTest(tf.test.TestCase):
-    def testRunInBackground(self):
-        mock_target = tf.compat.v1.test.mock.Mock()
-        with tf.compat.v1.test.mock.patch("threading.Thread", MockThread):
-            debug_data_multiplexer.run_in_background(mock_target)
-            mock_target.assert_called_once()
+    def testRunInBackgroundRepeatedly(self):
```

Comment:
nit, but would be better to have the test method name match the actual function (so testRunRepeatedlyInBackground)

Also, existing code, but a bit odd that this test class is named DebuggerV2PluginTest when this is in `debug_data_multiplexer_test.py`.

Reply:
Renamed the test class and test method as a part of rewriting these tests for the new signature of `run_repeatedly_in_background()`.